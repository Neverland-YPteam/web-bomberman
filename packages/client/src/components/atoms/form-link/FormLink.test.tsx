import React from 'react'
import { create } from 'react-test-renderer'
import { FormLink } from './index'
import { BrowserRouter } from 'react-router-dom'

const props = {
  to: 'link',
  text: 'test link'
}

it('MainMenuItem snapshot', () => {
  const actualSnapshot = create(
    <BrowserRouter>
      <FormLink to={props.to} text={props.text} />
    </BrowserRouter>).toJSON()

  expect(actualSnapshot).toMatchSnapshot()
})
