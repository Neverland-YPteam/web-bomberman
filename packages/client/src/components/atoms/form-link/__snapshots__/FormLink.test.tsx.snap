// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainMenuItem snapshot 1`] = `
<a
  className="css-1md0id3"
  href="/link"
  onClick={[Function]}
>
  test link
</a>
`;
